js/src/dag/put.js

```diff
@@ -113,6 +113,10 @@ module.exports = (createCommon, options) => {
       })
     })
 
+    it('shouldn\'t fail when calling put without options', (done) => {
```

Comment:
For consistency could you please use "should not" over "shouldn't"

Reply:
Done 👍